pkg/comp-functions/functions/vshnpostgres/maintenance.go

```diff
@@ -129,6 +130,11 @@ func addSchedules(ctx context.Context, comp *vshnv1.VSHNPostgreSQL, svc *runtime
 		},
 	}...)
 
+	err = setPsqlMinorVersion(svc, cluster)
```

Comment:
what if you set the version to null? Will the operator figure it out itself?

Reply:
Right now the version field is not required, so not setting it might be a way to go.

However, if they ever decide to make it required, it would break again. Then we'd have to go back to this solution.